doc/adoc/user_guide.adoc

```diff
@@ -90,6 +90,19 @@ If your system is not configured for it, it is recommended that at least
 one of the users on the system has a `~/.ssh/authorized_keys` file with a
 private key accessible by the person executing the system upgrade.
 
+== Upgrade Pre-Checks
+The suse-migration-pre-checks package contains the `suse-migration-pre-checks`
+script that checks for possible incompatibilities when doing a migration.
+
+These incompatibilities include:
+
+ Encrypted file systems
+ Invalid repository types
+
+This script is run during the install of suse-migration-pre-checks. It can
```

Comment:
Per discussion we will not run the prechecks here, but from the package that contains the ISO.

Reply:
Thanks I updated it to SLE15-Migration